### PR TITLE
Add confirmation number

### LINKED ISSUE #106: need confirmation number to give to frontend
i'm thinking about adding a uuid column to education_benefits_claims and using that for the confirmation number, thoughts @markolson?

### PR DESCRIPTION
**Pull request by lihanli**
fixes #106

### CODE REVIEW

File: spec/models/education_benefits_claim_spec.rb [hunk 1]
```diff
@@ -95,4 +95,11 @@
       expect(subject.regional_office).to eq("Eastern Region\nVA Regional Office\nP.O. Box 4616\nBuffalo, NY 14240-4616")
     end
   end
+
+  describe '#confirmation_number' do
+    it 'should return the right confirmation number' do
+      subject.save!
+      expect(subject.confirmation_number).to eq("vets_gov_education_benefits_claim_#{subject.id}")
```
Comment (carlmjohnson) [inline spec/models/education_benefits_claim_spec.rb]: This feels like the test is just a replication of the code. What if you tested that you can look up the claim number and get back the same claim?

File: app/models/education_benefits_claim.rb [hunk 2]
```diff
@@ -37,6 +37,10 @@ def parsed_form
     @parsed_form ||= JSON.parse(form)
   end
 
+  def confirmation_number
+    "vets_gov_#{self.class.to_s.underscore}_#{id}"
```
Comment (austin-martinez) [inline app/models/education_benefits_claim.rb]: Does this get displayed to the user? Is there any reason not to create a random uuid for this? My assumption is that this is used to communicate with the call center, and I would be concerned about giving a sequential identifier without knowing what other checks the call center does to verify identity

Comment (markolson) [inline app/models/education_benefits_claim.rb]: In an email, Shay wrote:
> The confirmation number is displayed to the user when they submit. It is just used to trace the application, if needed. (We can pull up the application in VONAPP Admin based on the confirmation number).
 
> It has no effect on the processing of the claim.

I'll ask for some followup on what "trace" means.

Comment (markolson) [inline app/models/education_benefits_claim.rb]: There's an Admin Interface in VONAPP where admins can look up forms by submission ID, and print copies of them out. The call center just passes those confirmation numbers along to someone with access to that admin interface. We can use basically whatever we want, but we also may need to make some sort of 'retransmit' mechanism to send the 22-1990 to a different processing center.

**Comment by lihanli** [comment]
@markolson which documentation?

File: docs/api/edu-benefits.md [hunk 3]
```diff
@@ -42,7 +42,8 @@ Content-Type: application/json
       // when it was first submitted
       "submittedAt": "2016-09-09T23:48:07.766Z",
       // address of the VA regional office where it will be processed
-      "regionalOffice": "Eastern Region\nVA Regional Office\nP.O. Box 4616\nBuffalo, NY 14240-4616"
+      "regionalOffice": "Eastern Region\nVA Regional Office\nP.O. Box 4616\nBuffalo, NY 14240-4616",
+      "confirmationNumber": "vets_gov_education_benefits_claim_18"
```
Comment (markolson) [inline docs/api/edu-benefits.md]: @lihanli this is the documentation I mentioned in the view

**Pull request merged**